org-formation/600-access/_tasks.yaml

```diff
@@ -181,25 +181,27 @@ CostExplorerAccessPolicy:
     Account: !Ref MasterAccount
     Region: !Ref primaryRegion
   Parameters:
-    PolicyDocument: '{
-      "Version": "2012-10-17",
-      "Statement": [
-        {
-          "Sid": "CostExplorerAccess",
-          "Effect": "Allow",
-          "Action": [
-              "aws-portal:ViewUsage",
-              "ce:Describe*",
-              "ce:Get*",
-              "ce:List*",
-              "ce:*AnomalyMonitor",
-              "ce:*AnomalySubscription",
-              "ce:*Report"
-          ],
-          "Resource": "*"
-        }
-      ]
-    }'
+    PolicyDocument:
+      !Sub
+        '{
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Sid": "${policyCostExplorerSid}",
```

Comment:
where is `policyCostExplorerSid` defined for this reference?

Reply:
I'm adding it to `_parameters.yaml`: https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/616/files#diff-69957a1b2df5fa15bc2b9dc2bb4c2321375dcf5966a15b43600657e1c34c7f12R24

Reply:
ahh, ok.  have you tested this with `org-formation print-task`?

Reply:
yeah, `npm run print-tasks-failfast` is happy